hw/ip/aes/data/aes.hjson

```diff
@@ -7,9 +7,56 @@
   name: "aes",
   clock_primary: "clk_i",
   bus_device: "tlul",
-  # Note: All parameters are local, they are not actually configurable.
-  # Selecting values different from the default values below might cause undefined behavior.
   param_list: [
+    { name:    "AES192Enable",
+      type:    "bit",
+      default: "1'b1",
+      desc:    '''
+        Disable (0) or enable (1) support for 192-bit key lengths (AES-192).
+      '''
+      local:   "false",
+      expose:  "false"
+    },
+    { name:    "Masking",
+      type:    "bit",
+      default: "1'b0",
+      desc:    '''
+        Disable (0) or enable (1) first-order masking of the AES cipher core.
+        Masking requires the use of a masked S-Box, see SBoxImpl parameter.
+      '''
+      local:   "false",
+      expose:  "true"
+    },
+    { name:    "SBoxImpl",
+      type:    "aes_pkg::sbox_impl_e",
+      default: "aes_pkg::SBoxImplLut",
+      desc:    '''
+        Selection of the S-Box implementation. See aes_pkg.sv.
+      '''
+      local:   "false",
+      expose:  "true"
+    },
+    { name:    "SecStartTriggerDelay",
+      type:    "int unsigned",
+      default: "0",
+      desc:    '''
+        Manual start trigger delay, useful for SCA measurements.
+        A value of e.g. 40 allows the processor to go into sleep before AES starts operation.
+      '''
+      local:   "false",
+      expose:  "true"
+    },
+    { name:    "AlertAsyncOn",
+      type:    "logic [aes_pkg::NumAlerts-1:0]",
+      default: "{aes_pkg::NumAlerts{1'b1}}",
+      desc:    '''
+        Bit array specifying whether a certain alert sender/receiver pair goes across an asynchronous boundary (1) or not (0).
```

Comment:
Would it make sense to be slightly more specific about what sender/receiver pair this is?

Reply:
I tried to make the description more clear.